apps/ewallet/lib/ewallet/web/v1/overlays/account_overlay.ex

```diff
@@ -80,4 +80,7 @@ defmodule EWallet.Web.V1.AccountOverlay do
       api_keys: APIKeyOverlay.self_filter_fields(),
       memberships: MembershipOverlay.self_filter_fields()
     ]
+
+  def pagination_fields,
```

Comment:
we should have `created_at` (inserted_at)  and `updated_at` for all resources as well